server/webapp/WEB-INF/rails.new/spec/views/admin/environment_variables_form_example.rb

```diff
@@ -57,3 +57,49 @@
     end
   end
 end
+
+shared_examples_for :secure_environment_variables_form do
+
+  it "should display the secure variables section" do
+    render :template => @view_file
+
+    expect(response.body).to have_selector("h3", :text => "Secure Variables");
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Do not use semicolons to terminate expressions.